app/models/feedback.rb

```diff
@@ -0,0 +1,4 @@
+class Feedback < ApplicationRecord
```

Comment:
Since your object replaces ratings and reviews you should just clean up those unused objects as well.